tests/main/views/test_direct_award.py

```diff
@@ -1049,13 +1049,18 @@ def test_invalid_data_raises_400_and_has_validation_messages_but_remains_in_form
         assert res.status_code == 400
 
         doc = html.fromstring(res.get_data(as_text=True))
-        assert doc.xpath('count(//span[@class="validation-message"])') == 1
+        assert (doc.xpath('count(//span[@class="validation-message"])') == 1 or
+                doc.xpath('count(//span[@class="govuk-error-message"])') == 1)
         errors = doc.cssselect('div.govuk-error-summary a')
         assert len(errors) == 1
         assert len(errors[0].text_content()) > 0
 
         for field, value in data.items():
-            assert doc.xpath(f'//input[@name="{field}"]/@value')[0] == value
+            input_element = doc.xpath(f'//input[@name="{field}"]')[0]
+            if (value is None or value == ''):
```

Comment:
I think this is equivalent and more concise:
```suggestion
            if not value:
```

Reply:
Thanks!